src/index.scss

```diff
@@ -61,17 +61,14 @@
 	width: 320px;
 }
 
-/* The two selectors below hide the inline inserters shown inside the block list */
-.block-editor-block-list__layout .block-editor-inserter {
-	display: none;
-}
```

Comment:
Removed to allow selecting, traversing, and modifying group blocks via the inline block appender.